.github/workflows/build.yml

```diff
@@ -74,3 +74,10 @@ jobs:
             -
                 name: Run bundle tests
                 run: composer test
+
+            -
+                name: Run tests for grids with php config
+                if: matrix.symfony == '^5.3'
+                run: |
+                    (cd src/Bundle/test && bin/console cache:clear --env=test_grids_with_php_config)
+                    APP_ENV=test_grids_with_php_config composer test
```

Comment:
I'm wondering if it is enough to test this new config style. The problem is that even tho, this may check the current implementation, we cannot be 100% sure. What I mean, is that you may remove the new config and the test will still pass (due to YAML configuration loaded in services.yaml). What we could do, is to have two envs `test_grids_with_php_config` and `test_grids_with_yaml_config`.